docs/CHANGELOG.rst

```diff
@@ -37,6 +37,8 @@ Changed
   when permissions to it are gained/lost or it is moved between containers. The
   notifications also contain the ``source`` and ``source_id`` context containing
   content type and id of the object that triggered the change.
+- Replace ``duplicate`` methods on ``Data``, ``Entity`` and ``Collection`` objects with
```

Comment:
This is a breaking change, as the client has to wait for the task to complete.

Reply:
Fixed in a changelog.